tests/core-providers/README.md

```
@@ -0,0 +1,338 @@
+
+Enable debug logging to see detailed API interactions:
+
+```bash
+go test -v ./tests/core-providers/ -debug
+```
+
+### Checking Provider Status
+
+If a provider seems to be failing, you can check their status pages:
+
+- OpenAI: https://status.openai.com/
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)